pkgconfig/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# CMakeLists.txt for libyui-mga/pkgconfig
+#
+# Support for pkg-config:
+#
+# Generate a libyui.pc file from libyui.pc.in and install it to
+# /usr/lib64/pkgconfig.
+#
+# A .pc file specifies how to use a development package, in particular linker
+# flags (-lyui-gtk), compiler flags (including include directories) and paths.
+# See man pkg-config.
+#
+# Notice that libyui-mga linking is needed for libyui-mga-* plugins 
+
+include( ../VERSION.cmake )
+include( GNUInstallDirs )       # set CMAKE_INSTALL_LIBDIR
+
+FIND_PACKAGE(PkgConfig REQUIRED)
+
+PKG_CHECK_MODULES(YUI REQUIRED libyui)
+pkg_get_variable(YUI_SO_VERSION libyui soversion)
```

Comment:
Is any of those `YUI_SO_VERSION` etc.  needed? I don't see it used in libyui-gtk.pc.in. To me it looks as if lines 20-26 can safely be deleted.

Reply:
added them as said above:

```
soversion_major=@SONAME_MAJOR@
soversion_minor=@SONAME_MINOR@
soversion_patch=@SONAME_PATCH@
soversion=@SONAME@
```